components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/SAMLSSORelyingPartyObject.java

```diff
@@ -403,104 +403,31 @@ public static String jsFunction_getUUID(Context cx, Scriptable thisObj,
-                }
-            }
-        } else {
-            //When number of args are not eq. to 2, then its assumed to be a non passive auth request build
-            if (!Boolean.valueOf(relyingPartyObject.getSSOProperty(SSOConstants.SIGN_REQUESTS))) {
-                //builds an unsigned non-passive authentication request
-                return Util.marshall(new AuthReqBuilder().
-                        buildAuthenticationRequest(
-                                relyingPartyObject.getSSOProperty(SSOConstants.ISSUER_ID), nameIdPolicy));
-            } else {
-                if (argLength == 0) {
-                    //builds a signed non-passive authentication request without consumer url
-                    return Util.marshall(new AuthReqBuilder().
-                            buildSignedAuthRequest(
-                                    relyingPartyObject.getSSOProperty(SSOConstants.ISSUER_ID),
-                                    relyingPartyObject.getSSOProperty(SSOConstants.IDP_URL),
-                                    MultitenantConstants.SUPER_TENANT_ID,
-                                    MultitenantConstants.SUPER_TENANT_DOMAIN_NAME, nameIdPolicy));
-                } else {
-                    //builds a signed non-passive authentication request with consumer url
-                    String consumerUrl = (String) args[0];
-                    return Util.marshall(new AuthReqBuilder().
-                            buildSignedAuthRequestWithConsumerUrl(
-                                    relyingPartyObject.getSSOProperty(SSOConstants.ISSUER_ID),
-                                    relyingPartyObject.getSSOProperty(SSOConstants.IDP_URL),
-                                    consumerUrl, MultitenantConstants.SUPER_TENANT_ID,
-                                    MultitenantConstants.SUPER_TENANT_DOMAIN_NAME, nameIdPolicy));
-                }
-            }
-        }
-    }
-
-    /**
-     * Provides a way to includes passiveAuth and assertionConsumerServiceURL parameters to SAML Auth request
-     *
-     * @param cx
-     * @param thisObj
-     * @param args - args[0] indicates passiveAuth required or not args[1] indicates assertionConsumerServiceURL
-     * @param funObj
-     * @return
-     * @throws Exception
-     */
-
-    public static String jsFunction_getSAMLPassiveAuthRequest(Context cx, Scriptable thisObj,
-                                                              Object[] args,
-                                                              Function funObj) throws Exception {
-        SAMLSSORelyingPartyObject relyingPartyObject = (SAMLSSORelyingPartyObject) thisObj;
-        String nameIdFormat = relyingPartyObject.getSSOProperty(SSOConstants.NAME_ID_POLICY);
 
-        if ((Boolean) args[0])  {
-            String acsUrl = (String) args[1];
+        //check if request signing is required
+        if (!Boolean.valueOf(relyingPartyObject.getSSOProperty(SSOConstants.SIGN_REQUESTS))) {
```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/0e5a93cca133738cc74a26863424e20db485892d)